src/fides/api/service/connectors/query_config.py

```diff
@@ -810,14 +813,41 @@ class BigQueryQueryConfig(QueryStringWithoutTuplesOverrideQueryConfig):
     Generates SQL valid for BigQuery
     """
 
+    def __init__(
+        self,
+        node: ExecutionNode,
+        namespace_meta: Optional[BigQueryNamespaceMeta] = None,
```

Comment:
Updating the init to include an optional `namespace_meta` object

Reply:
This is valid, I went ahead and made this change. Let me know what you think of my implementation